website/pages/docs/k8s/crds.mdx

```diff
@@ -0,0 +1,482 @@
+  expose:
+    checks: true
+    paths:
+      - listenerPort: 80
+        localPathPort: 42
+        path: /test/path
+        protocol: tcp
+      - listenerPort: 8080
+        localPathPort: 4201
+        path: /root/test/path
+        protocol: https
+  meshGateway:
+    mode: local
+```
+
+Notes:
+
+1. There can only be one `ProxyDefaults` resource and its name **must** be `global`.
+1. Keys under `spec.config` will need to use `snake_case`, not `camelCase`, because
+   this config is passed directly to Envoy which uses `snake_case`, e.g.
+   ```yaml
+   spec:
+     config:
+       envoy_statsd_url: 'udp://127.0.0.1:8125'
+   ```
+1. See [proxy-defaults](/docs/agent/config-entries/proxy-defaults) for full
+   documentation on this config entry.
+
+### ServiceDefaults
+
+[service-defaults](/docs/agent/config-entries/service-defaults)
+configures a specific service.
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceDefaults
+metadata:
+  name: your-service-name
+spec:
+  protocol: https
+  meshGateway:
+    mode: local
+  expose:
+    checks: true
+    paths:
+      - listenerPort: 80
+        path: /path
+        localPathPort: 9000
+        protocol: tcp
+      - listenerPort: 8080
+        path: /another-path
+        localPathPort: 9091
+        protocol: http2
+  externalSNI: external-sni
+```
+
+Notes:
+
+1. The name of the service that this `ServiceDefaults` configures is set by
+   `metadata.name`.
+1. See [service-defaults](/docs/agent/config-entries/service-defaults) for full
+   documentation on this config entry.
+
+### ServiceIntentions <sup>(Beta)</sup>
+
+[service-intentions](/docs/agent/config-entries/service-intentions)
+configures traffic authorization for a destination service.
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceIntentions
+metadata:
+  name: my-intention
+spec:
+  destination:
+    name: your-service-name
+  sources:
+    - name: svc1
+      namespace: test
+      action: allow
+      permissions: []
+      description: allow access from svc1
+    - name: '*'
+      namespace: not-test
+      action: deny
+      permissions: []
+      description: disallow access from namespace not-test
+    - name: svc-2
+      namespace: bar
+      action: ''
+      permissions:
+        - action: allow
+          http:
+            pathExact: /foo
+            pathPrefix: /bar
+            pathRegex: /baz
+            header:
+              - name: header
+                present: true
+                exact: exact
+                prefix: prefix
+                suffix: suffix
+                regex: regex
+                invert: true
+            methods:
+              - GET
+              - PUT
+      description: an L7 config
+```
+
+Notes:
+
+1. This resource is only supported in Consul 1.9.0.
+1. Unlike the other configuration entries, the value of `metadata.name` does not
```

Comment:
Maybe explain why? It's implied by the nature of the entry, but it never hurts to spell things out.

```suggestion
1. **Unlike the other configuration entries**, the value of `metadata.name` does not
```